content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -306,6 +313,60 @@ few times, after a while you should see the spans printed in the console by the
   "status": { "code": 1 },
   "events": []
 }
+```
+
+</details>
+
+The span generated for you tracks the lifetime of a request to the `/rolldice`
+route.
+
+Send a few more requests to the endpoint, and then wait for a little bit and
+you'll get metrics printed out to the console, such as the following
+
+<details>
+<summary>View example output</summary>
+
+```javascript
```

Comment:
```suggestion
```json
```

Reply:
maybe there is a jsonlines option?

Reply:
@svrnm, maybe try `jsonlines` (I'd never tried that one before). If it doesn't work go with which ever option you prefer.

Reply:
I put in `jsonlines`, also not sure if this is a legal argument although it is not screaming at me in red anymore in VSCode, I would say that's a win